src/maelstrom/workload/lock.clj

```diff
@@ -0,0 +1,79 @@
+(ns maelstrom.workload.lock
+  "A distributed lock workload: sends lock and unlock messages."
```

Comment:
It'd be helpful here to describe lock semantics. I'm guessing this workload assumes at most one client holds a lock at a time (and specifically that the lock is linearizable), and that there's no recovery for crashed or timed-out clients?

Reply:
Yes, this is a very basic workload, for a single lock object (no instances, no keys), assumes no crashed clients and that clients do not timeout (see below).